tests/hypothesis_strategies/targets.py

```diff
@@ -0,0 +1,34 @@
+"""Hypothesis strategies for targets."""
+
+import hypothesis.strategies as st
+from attrs import NOTHING
+
+from baybe.targets.enum import TargetMode
+from baybe.targets.numerical import _VALID_TRANSFORM_MODES, NumericalTarget
+
+from .utils import interval
+
+target_name = st.text(min_size=1)
+"""A strategy that generates target names."""
+
+
+@st.composite
+def numerical_target(draw: st.DrawFn):
+    """Generate class:`baybe.targets.numerical.NumericalTarget`."""
+    name = draw(target_name)
+    mode = draw(st.sampled_from(TargetMode))
+    bounds = draw(
+        interval(exclude_half_open=True, exclude_open=mode is TargetMode.MATCH)
```

Comment:
Again, why the exclusion?

Reply:
we cant handle any half-bounded intervals anywhere at the moment hence the exclusion form this strategy (that is supposed to create valid intervals)

Reply:
Alright, I think everything's clear now. Will resolve and merge as soon as the checks have passed 🚀